index.js

```diff
@@ -64,31 +54,43 @@ var FocusTrap = React.createClass({
     } else if (!prevProps.paused && this.props.paused) {
       this.focusTrap.pause();
     }
-  },
+  }
 
-  componentWillUnmount: function() {
+  componentWillUnmount() {
     this.focusTrap.deactivate();
     if (this.props.focusTrapOptions.returnFocusOnDeactivate !== false && this.previouslyFocusedElement) {
       this.previouslyFocusedElement.focus();
     }
-  },
+  }
 
-  render: function() {
-    var props = this.props;
+  render() {
+    const props = this.props;
```

Comment:
why not just spread them and return jsx rather than via `createElement` API

```jsx
  render() {
    const props = this.props;
    const elementProps = {}

    // This will get id, className, style, etc. -- arbitrary element props
    for (const prop in props) {
      if (props.hasOwnProperty(prop) && !(prop in checkedProps)) {
        elementProps[prop] = props[prop];
      }
    }

    return <props.tag {...elementProps} ref={e => this.node = e}>{props.children}</props.tag>
  }
```